chapters/functions.tex

```diff
@@ -1414,6 +1414,12 @@ \subsection{Using the Derivative Annotation}\label{using-the-derivative-annotati
   //annotation(derivative(order=2) = f_general_der2);
 end f_general_der;
 \end{lstlisting}
+In the example above \lstinline!zeroDerivative=y! implies that
+\begin{eqnarray*}
+\frac{d}{dt}f(x(t),y(t))&=&\frac{\partial f}{\partial x}\frac{dx}{dt}+\frac{\partial f}{\partial y}\frac{dy}{dt}+\frac{\partial f}{\partial o}\frac{do}{dt}\\
+&=&\frac{\partial f}{\partial x}\frac{dx}{dt}+\frac{\partial f}{\partial y}\cdot 0+\frac{\partial f}{\partial o}\cdot 0\\
+&=&\frac{\partial f}{\partial x}\frac{dx}{dt}
```

Comment:
Should we include `f_der` into the formula? 
```suggestion
&=&\frac{\partial f}{\partial x}\frac{dx}{dt}\\
&=&f\_der
```
Or it can be on the same line?

Reply:
I would prefer not to include `f_der` at this time, as the underscore looks misplaced in the pdf-file. 
The next option would be to rename f_der in this case as well (e.g. to h) - which should work although a bit odd.

Reply:
I renamed to fDer, and included it.